### DISCUSSION TITLE
Is there a client library now?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nanoframework/Home#1029] [General]
### Description

Is there a client library now?

I used to use this development and found that there was no ble client library. I gave it up for some time. Recently, there is a demand. I don't know if I have it now? I needed this library and had to give it up and use Arduino instead.

### How to solve the problem

I used to use this development and found that there was no ble client library. I gave it up for some time. Recently, there is a demand. I don't know if I have it now? I needed this library and had to give it up and use Arduino instead.

**Comment by josesimoes** [comment]
This is not a bug, moving to discussion.

**Comment by josesimoes** [comment]
Client for??... 😯

**Reply by alberk8** [reply]
Most likely looking for the Client/Central BLE mode, which is already a [WIP](https://github.com/nanoframework/Home/issues/981).

**Reply by bbhxwl** [reply]
> Most likely looking for the Client/Central BLE mode, which is already a [WIP](https://github.com/nanoframework/Home/issues/981).

yes
